### DISCUSSION TITLE
Comments, Mentions, and References

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4792] [Feature Requests]
* [x] Mentions Dropdown on `@`
    * [x] If a user is mentioned in a comment, the user should be notified by email of it
* [x] Reference Dropdown on `#` or `@`?
* [ ] Emoji inline dropdown on `:xx` (at least two characters)
* [x] Force open all external markdown links in new tab
* [x] Emails/Notifications for Comment Mentions
* [ ] Track “Seen” (requires dedicated system collection)

**Comment by workatease** [comment]
@benhaynes Any Plans to separate the comments out of activity tracker and adding Track "Seen"

**Reply by benhaynes** [reply]
I think adding the 'seen' feature is something we'll definitely want to do (when we have time/bandwidth)... but not sure about the need to separate those collections... @rijkvanzanten ?

**Reply by rijkvanzanten** [reply]
Notifications have an archive state already. That's not currently automatically set though. As for splitting comments from activity, we could, though there's currently no plan to do so (happy to discuss in a new feature request if you have a specific need/reason for that 👍🏻 )

**Reply by workatease** [reply]
Hi @rijkvanzanten , The reason I think the collection should be split is that it,
As of now the the activity collection flood in a matter of days when we have a there are multiple users 
And the use case is there will 
1.Each user will have a data entered in a collection 
 2. reviewers of the data will check and add comments for the data with notification to multiple users to correct/fix the entered data,
 3. Inspected with approved it with comments / or rejects with comments 
 4. and the process will loop to the tep 2 to 4 multiple time .

with min 2000 Rows in each collection and 15 collections
so on this happens for 10 of my collection and when this happens with 100 users and 30 approvers 
activity collection fills up very fast and the comments are hidden in the activity collection.

**Reply by workatease** [reply]
I think it would be good idea to split the collection and have the ability to mark it notification as read the same way in github notification
